src/gltf-component-mappings.js

```diff
@@ -4,6 +4,9 @@ AFRAME.GLTFModelPlus.registerComponent("quack", "quack");
 AFRAME.GLTFModelPlus.registerComponent("sound", "sound");
 AFRAME.GLTFModelPlus.registerComponent("collision-filter", "collision-filter");
 AFRAME.GLTFModelPlus.registerComponent("css-class", "css-class");
+AFRAME.GLTFModelPlus.registerComponent("interactable", "css-class", (el, componentName) => {
```

Comment:
Why not actually just use the css class component?

Reply:
Spoke cannot export components with literal prop values. Prop values are always objects.